docker-compose.yml

```
@@ -529,6 +530,10 @@ volumes:
     external: true
   sentry-symbolicator:
     external: true
+  # This volume stores JS SDK assets, this is created as an external volume
+  # to minimize hiccups created by the nginx container and its' dependencies.
+  # The data inside this volume should be cleaned periodically on upgrades.
```

Just helping you out :)

```suggestion
  # This volume stores JS SDK assets and the data inside this volume should
  # be cleaned periodically on upgrades.
```